apps/website/src/pages/index.astro

```diff
@@ -435,4 +445,28 @@ const currentPage = new URL(Astro.request.url).pathname;
       }
     }
   </style>
+
+  <script>
+    const updateVisibility = () => {
+      const visibleBottomLine = window.scrollY + window.innerHeight;
+      const demoWrappers = document.querySelectorAll('.move-on-visible');
+      demoWrappers.forEach((demoWrapper) => {
+        if (!(demoWrapper instanceof HTMLElement)) {
+          return;
+        }
+
+        const top = demoWrapper.getBoundingClientRect().top;
+        if (top < visibleBottomLine) {
+          demoWrapper.dataset.visible = 'true';
```

Comment:
`data-visible` is too generic and might already be used in other places. maybe scope this as `data-move-visible` or something

Reply:
Changed `data-visible` to `data-move-visible`